dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops

```diff
@@ -49,4 +49,8 @@ CWMOT5003.factory.used.after.shutdown.useraction=Review the application to see w
 
 CWMOT5004.factory.used.without.metadata=CWMOT5004W: A request was made for an OpenTelemetry object but no application metadata, and no global OpenTelemetry object was available. 
 CWMOT5004.factory.used.without.metadata.explanation=CWMOT5004.factory.used.without.metadata.explanation=If there is no global OpenTelemetry object Liberty's MicroProfile Telemetry integration code will attempt to find the correct OpenTelemetry object for the current application by using the application metadata on the current thread. However as there was no metadata on the thread it was unable to do so. 
-CWMOT5004.factory.used.without.metadata.useraction=Review your code to see why it attempted to use MicroProfile Telemetry when there was no application metadata and server scoped OpenTelemetry object. The most likely case is that you are creating new threads without using Liberty's ManagedExecutorService and trying to acquire an OpenTelemetry object from within. If you are attempting to acquire an OpenTemetry object outside an application, ensure OpenTelemetry is enabled via an environment variable or system property.   
\ No newline at end of file
+CWMOT5004.factory.used.without.metadata.useraction=Review your code to see why it attempted to use MicroProfile Telemetry when there was no application metadata and server scoped OpenTelemetry object. The most likely case is that you are creating new threads without using Liberty's ManagedExecutorService and trying to acquire an OpenTelemetry object from within. If you are attempting to acquire an OpenTemetry object outside an application, ensure OpenTelemetry is enabled via an environment variable or system property.   
+
+CWMOT5005.mptelemetry.unknown.log.source=CWMOT5005W: MicroProfile Telemetry Logging feature does not recognize the specified {0} log source.
+CWMOT5005.mptelemetry.unknown.log.source.explanation=The specified log source is not one of the log sources provided by the Liberty runtime environment.
```

Comment:
The Liberty runtime environment does not provide the specified log source.

Reply:
Fixed.